spec/lib/que/job_spec.rb

```diff
@@ -87,6 +94,28 @@
       end
     end
 
+    context "with keyword args" do
+      let(:job_class) do
+        Class.new(described_class) do
+          def run(name:, password:)
+            true
+          end
+        end
+      end
+
+      it "correctly serializes and deserializes the arguments" do
+        job = job_class.enqueue(name: "Tony", password: "tiger")
+        job._run
+      end
+
+      context "missing the required argument" do
+        it "returns an ArgumentError" do
+          job = job_class.enqueue(name: "Tony")
+          expect { job._run }.to raise_error(ArgumentError, /missing keyword.+password/)
```

Comment:
Regex needed because this message changed between Ruby 2.6 and 2.7 🤦

Reply:
Maybe worth adding this as a quick comment?